static/src/javascripts/projects/commercial/modules/dfp/create-slot.ts

```diff
@@ -179,14 +181,24 @@ const createAdSlotElement = (
 	// The 'main' adSlot
 	const adSlot = document.createElement('div');
 	adSlot.id = id;
-	adSlot.className = `js-ad-slot ad-slot ${classes.join(' ')}`;
+	adSlot.className = `js-ad-slot ad-slot ${
+		includeContainer ? '' : classes.join(' ')
+	}`;
```

Comment:
We need to move these classes up one level to the container otherwise it messes up the styling.

Note: `classes.join(' ')` evaluates to `ad-slot--inline ad-slot--offset-right ad-slot--inline2` for the `inline2` ad, for example